src/Components/Components/src/RenderTree/RenderTreeFrameType.cs

```diff
@@ -6,7 +6,7 @@ namespace Microsoft.AspNetCore.Components.RenderTree
     /// <summary>
     /// Describes the type of a <see cref="RenderTreeFrame"/>.
     /// </summary>
-    public enum RenderTreeFrameType: int
+    public enum RenderTreeFrameType: short
```

Comment:
Strictly speaking this change isn't required in this PR, but I'd like to include it because it's a good thing and means we have less to reason about in the future if we want to make use of the memory space this frees up.